### PR TITLE
pkg/kamailio/centos: Added SPEC file for CentOS7

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

**Comment by sergey-safarov** [comment]
@linuxmaniac thanks for fast review.
Prefix added.

**Comment by sergey-safarov** [comment]
Hello @linuxmaniac
Why not use kamailio.service and kamailio.sysconfig from debian?
This is more easy to maintain packaging.

About difference of environment file location. This is adjusted in kamailio.spec:595

https://github.com/kamailio/kamailio/pull/966/files#diff-7757a46a5c00eaeca332f4a6be46e96fR595

File: pkg/kamailio/centos/7/kamailio.spec [hunk 1]
```diff
@@ -0,0 +1,1387 @@
...
+
+%description    json
+json string handling and RPC modules for Kamailio.
+
+
+%if %{with kazoo}
+%package    kazoo
+Summary:    Kazoo middle layer connector support for Kamailio
+Group:      System Environment/Daemons
+Requires:   libuuid, kamailio = %ver
```
Comment (lazedo) [inline pkg/kamailio/centos/7/kamailio.spec]: missing rabbitmq-c, json-c, libevent

File: pkg/kamailio/centos/7/kamailio.spec [hunk 2]
```diff
@@ -0,0 +1,1387 @@
...
+BuildRequires:	rabbitmq-c-devel, json-c-devel, libuuid-devel
```
Comment (lazedo) [inline pkg/kamailio/centos/7/kamailio.spec]: missing libevent-devel

**Comment by sergey-safarov** [comment]
@lazedo I will update master branch after this PR is merged.

**Comment by linuxmaniac** [comment]
> Why not use kamailio.service and kamailio.sysconfig from debian?
> This is more easy to maintain packaging.

because is what we have in master and from my point of view is better to be independient

**Comment by sergey-safarov** [comment]
Hello @linuxmaniac 
Pull request updated.

**Pull request merged**